src/backend/dx11/src/lib.rs

```diff
@@ -709,7 +711,10 @@ pub struct CommandBuffer {
     #[derivative(Debug="ignore")]
     context: ComPtr<d3d11::ID3D11DeviceContext>,
     #[derivative(Debug="ignore")]
-    list: Option<ComPtr<d3d11::ID3D11CommandList>>
+    list: Option<ComPtr<d3d11::ID3D11CommandList>>,
+    vertex_buffers: [*mut d3d11::ID3D11Buffer; 16],
```

Comment:
I think I already asked somewhere but forgot the answer - why aren't we using `ComPtr` here and in descriptors? Is this to save the clocks of bumping the refcounts? Doesn't appear too important for DX11 in this case (but still makes sense for DX12 if applicable).

Reply:
Mostly to avoid having to make an API call to set every resource individually when the functions take start, end, ptr arguments

Reply:
yep, makes sense